### ISSUE TITLE
Peer-to-peer code review

### ISSUE THREAD

**Issue by Zuwairanajma** [the-faizmohammad/my-portfolio#14]
Great job with the portfolio project.
 
I am impressed with your usage of a single stylesheet and your ability to style with minimum lines of code.
However I noticed some accessibility rules were not adhered to, please correct the header tags' semantics, they should be hierarchical. 
I strongly recommend you use the w3validator checker to make possible accessibility changes required.
